packages/sanity/test/validation/__snapshots__/numbers.test.ts.snap

```diff
@@ -194,7 +194,7 @@ Array [
   Object {
     "item": ValidationError {
       "children": undefined,
-      "message": "Value is required",
+      "message": "Required",
```

Comment:
❓ should this still be `Value is required` vs just `Required` ?

Reply:
I took the liberty of changing it - all the other validators were using simply "Required", so I felt it was too inconsistent to leave this one with a different message.